dev/benchmarks/complex_layout/test_driver/semantics_perf_test.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:path/path.dart' as p;
+import 'package:test/test.dart';
+
+void main() {
+  group('semantics performance test', () {
+    FlutterDriver driver;
+
+    setUpAll(() async {
+      driver = await FlutterDriver.connect(printCommunication: true);
+    });
+
+    tearDownAll(() async {
+      if (driver != null)
+        driver.close();
+    });
+
+    test('inital tree creation', () async {
+      // Let app become fully idle.
+      await new Future<Null>.delayed(new Duration(seconds: 1));
```

Comment:
I can't find any element to wait for, though. Shortly after app app loads the dimensions of some elements change, but I don't see a way to robustly wait for that...